### DISCUSSION TITLE
Recurring events

### DISCUSSION THREAD

**Discussion by langhimebaugh** [ics-py/ics-py#297] [Ideas]
The documentation states "At parsing time, every property or container that is unknown to ics.py (and thus not handled) is stored in .extra."  My ICS file contains RDATE for recurring events but these appear to be ignored.  Can't you place these in .extra so we can still access this info?

Seems like something is being done here...
https://github.com/ics-py/ics-py/blob/5ff40f88f3307515fe04e6ae5be0d70cee0d6f2e/src/ics/converter/types/various.py

Example of RDATE in the ICS file
DTSTART;TZID=America/New_York:20211222T220000
DTEND;TZID=America/New_York:20211222T220000
RDATE;TZID=America/New_York:20211223T220000
RDATE;TZID=America/New_York:20211224T220000

**Comment by N-Coder** [comment]
Hey, any unknown properties should land in `extra`. I implemented a parser for RRULES, but that is not yet used as `Event` has no such field, yet. Could you include a minimal example ics file for reproduction and the `repr` of the corresponding parsed calendar? Furthermore, are you using the release from PyPi or the current master branch (i.e. what does `ics.__version__` say)?
